community/views.py

```diff
@@ -8,21 +8,104 @@
     get_org_name,
     get_owner,
     get_upstream_deploy_url,
+    get_remote_url
 )
+from data.models import Team, Contributor
+
+
+def initialize_org_context_details():
+    org_name = get_org_name()
+    org_details = {
+        'name': org_name,
+        'blog_url': 'https://blog.{}.io/'.format(org_name),
+        'twitter_url': 'https://twitter.com/{}_io/'.format(org_name),
+        'facebook_url': 'https://www.facebook.com/{}Analyzer'.format(
+            org_name),
+        'repo_url': get_remote_url().href,
+        'docs': 'http://{}.io/docs'.format(org_name),
+        'newcomer_docs': 'http://{}.io/newcomer'.format(org_name),
+        'coc': 'http://{}.io/coc'.format(org_name),
+        'logo_url': ('https://api.{}.io/en/latest/_static/images/'
+                     '{}_logo.svg'.format(org_name, org_name)),
+        'gitter_chat': 'https://gitter.im/{}/{}/'.format(org_name,
+                                                         org_name),
+        'github_core_repo': 'https://github.com/{}/{}/'.format(org_name,
+                                                               org_name),
+        'licence_type': 'GNU AGPL v3.0'
+    }
+    return org_details
+
+
+def get_header_and_footer(context):
+    context['isTravis'] = Travis.TRAVIS
+    context['travisLink'] = Travis.TRAVIS_BUILD_WEB_URL
+    context['org'] = initialize_org_context_details()
+    print('Running on Travis: {}, build link: {}'.format(context['isTravis'],
+                                                         context['travisLink']
+                                                         ))
+    return context
 
 
 class HomePageView(TemplateView):
     template_name = 'index.html'
 
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        context['isTravis'] = Travis.TRAVIS
-        context['travisLink'] = Travis.TRAVIS_BUILD_WEB_URL
+    def get_team_details(self):
+        org_name = get_org_name()
+        teams = [
+            '{} newcomers'.format(org_name),
+            '{} developers'.format(org_name),
+            '{} admins'.format(org_name)
+        ]
+        team_details = {}
+        for team in teams:
+            team_objs = [Team.objects.get(name=team), ]
+            contributors = Contributor.objects.filter(teams__in=team_objs)
+            team_details[team] = contributors.count()
+        return team_details
+
+    def get_quote_of_the_day(self):
+        import requests
+        qod = requests.get('http://quotes.rest/qod?category=inspire')
+        qod_data = qod.json()
+        if qod.status_code == 429:
+            return None
+        return {
+            'quote': qod_data['contents']['quotes'][0]['quote'],
+            'author': qod_data['contents']['quotes'][0]['author'],
+        }
+
+    def get_top_meta_review_users(self, count):
+        from meta_review.models import Participant
```

Comment:
> Any specific reason to have the import here instead of at the top of the module?

No! Specific reason. I have seen in some of the upstream packages where they were also importing a module within a function.
Will Move it to top of the module